packages/observability/mock-data/import-mock-data.sh

```
@@ -0,0 +1,397 @@
+        oc port-forward -n "$PROMETHEUS_NAMESPACE" "svc/$PROMETHEUS_SERVICE" "$PROMETHEUS_PORT:9090" &
+        PF_PID=$!
+        sleep 3
+
+        # Verify port-forward is working
+        if ! kill -0 $PF_PID 2>/dev/null; then
+            echo "Error: Port-forward failed to start"
+            exit 1
+        fi
+
+        echo "Port-forward established (PID: $PF_PID)"
+        trap "kill $PF_PID 2>/dev/null" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)